test/run

```diff
@@ -45,7 +43,14 @@ container_exists() {
 }
 
 container_ip() {
-  docker inspect --format="{{ .NetworkSettings.IPAddress }}" $(cat $cid_file)
+  # local ip=$(docker inspect --format="{{ .NetworkSettings.IPAddress }}" $(cat $cid_file))
+  # echo ${ip:-localhost}
+  echo localhost
+}
+
+test_port() {
```

Comment:
This function could be a part of container-common-scripts.
The first parameter would be port and the second parameter would be a protocol

The results would be `echo` as you already have it.

Reply:
Sounds good. I will need to make time to get back to this soon and get this PR to a mergeable state.